warehouse/query-core/src/main/java/datawave/query/tables/facets/FacetedQueryLogic.java

```diff
@@ -173,14 +173,13 @@ public boolean apply(Entry<Key,Value> input) {
     
     @Override
     public Set<String> getOptionalQueryParameters() {
-        Set<String> params = new TreeSet<>();
-        params.add(FacetedConfiguration.MINIMUM_COUNT);
-        params.add(FacetedConfiguration.FACETED_SEARCH_TYPE);
-        params.add(FacetedConfiguration.FACETED_FIELDS);
-        params.add(FacetedConfiguration.MAXIMUM_GROUP_COUNT);
-        params.add(FacetedConfiguration.STREAMING_ENABLED);
-        params.add(QueryOptions.LIMIT_FIELDS);
```

Comment:
Was LIMIT_FIELDS dropped intentionally? It appears optional from a quick peek at the code.

Reply:
Nevermind, I overlooked the super call. Cool.